index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>판다마켓</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
+  <header>
+    <a href="/" class="logo click"> <img src="images/logo.png"> </a>
+    <div class="login">
+     <a class="click" href="/login"> 로그인 </a>      
+    </div>
```

Comment:
```suggestion
<a class="click" href="/login">
    <div class="login">
      로그인  
    </div>
</a>
```
현재 로그인 버튼 중 스크린샷에서 보라색 영역에 해당되는 부분은 커서의 모양도 변하지 않고 링크로 이동하는 동작로 이루어지지 않습니다 🥲 일반적으로는 `div`에 해당하는 영역 전체에 대해 동작을 기대할 것이므로 `a` 태그로 감싸는 것을 추천드립니다.
(이 경우 CSS 선택자 쪽에서도 수정이 필요합니다. 예를 들면 `.login .click`이런식으로 자손선택자로 되어있는 부분들이요!)

<img width="214" alt="스크린샷 2024-04-29 오전 9 33 42" src="https://github.com/codeit-bootcamp-frontend/7-Sprint-Mission/assets/144018522/9262b18f-1cf7-4110-ae33-b9c1a5117302">